addons/crm/report/crm_activity_report_views.xml

```diff
@@ -57,7 +57,7 @@
                     <separator/>
                     <filter string="Trailing 12 months" name="completion_date" domain="[
                         ('date', '>=', (datetime.datetime.combine(context_today() + relativedelta(days=-365), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('date', '&lt;=', (datetime.datetime.combine(context_today(), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
+                        ('date', '&lt;=', (datetime.datetime.combine(context_today(), datetime.time(23, 59, 59)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
```

Comment:
I don't know why but when I inspect the "Trailing 12 months" filter, I get this weird display saying the datetime isn't valid? :thinking: 
![image](https://github.com/user-attachments/assets/747be5c3-5a87-4c80-92c1-a2b1270fc1d9)

Reply:
Ah yes @amdi-odoo , it is also the case in master actually. The message is shown if no isSupported:

`isSupported: (value) => value === false || (typeof value === "string" && isParsable(type, value)),`

,with parser being parseDateTime() in JS.

This makes sense since this is not a regular date format, but rather an expression. The default warning message will be shown, I'm afraid :p